drivers/sx127x/sx127x_netdev.c

```diff
@@ -65,9 +65,7 @@ static int _send(netdev_t *netdev, const iolist_t *iolist)
             break;
         case SX127X_MODEM_LORA:
             /* Initializes the payload size */
-            if (!sx127x_get_fixed_header_len_mode(dev)) {
-                sx127x_set_payload_length(dev, size);
-            }
+            sx127x_set_payload_length(dev, size);
```

Comment:
Do we really need to re-write this register every time? If the user wants to go to implicit mode it is expected that it will use the `NETOPT_PDU_SIZE` option to set the payload (only once), then enable implicit mode and send/receive. Right?